### ISSUE TITLE
Add a button to the Mapillary field to set the value to the currently selected Mapillary photo id

### ISSUE THREAD

**Issue by RudyTheDev** [openstreetmap/iD#9339]
In addition, always show the [`mapillary`](https://wiki.openstreetmap.org/wiki/Key:mapillary) field when a Mapillary photo is selected.

This would significantly simplify this action. Currently, one has to either (A) open the photo link, copy the id from the browser's URL bar, and close the tab, add the field, and paste the id into the field or (B) copy the photo link, add the field, paste the id into the field, and trim the non-id part from the URL.

This should encourage more use and promote verifiability. It would also make mapping with Mapillary in iD a little more usable. Without going into tangentials, it's not the easiest workflow at the moment.

It would also make sense to have a button to activate Mapillary layer and select the photo. Not sure how viable this is though - in general, one would have to query Mapillary API for the location of the photo, so this wouldn't be an instant action. Perhaps it may need to be restricted to those Mapillary photos that are currently loaded/visible.

**Comment by timothywashere** [comment]
I also find it quite annoying that there is no easy way to add a Mapillary image tag to something when I am using the Mapillary layer inside the iD editor. I have to click on the mapillary icon to open the image in another window, then find the id and close the window again. It's such a waste of time and bandwidth.
So it would be very nice if there was a quick way of getting the iD of the currently displayed Mapillary image within the iD editor

**Comment by Ni-vik** [comment]
@tyrasd Is this issue still open? It's my first time contributing in openstreetmap as this issue is labelled with good first issue , I would like to work on this.

**Comment by tordans** [comment]
FYI, there is a comment of mine from Dez 2000 with some Wireframe-like illustrations of this feature idea https://github.com/openstreetmap/iD/issues/6196#issuecomment-751460516

**Comment by jdabapo** [comment]
@Ni-vik are you working on this issue already? I would like to contribute as well, but I do not want to take an issue away if you are already working on it.

**Comment by Ni-vik** [comment]
@jdabapo yep I am working on it

**Comment by kunjbosamia** [comment]
Hey is this issue still open if yes then I would like to work on this

**Comment by Gua00va** [comment]
Hey!! I want to work on this issue. I'm new here so I'm really excited to start contributing. Please assign it to me.

**Comment by KishorKumarParoi** [comment]
Hello Friends and folks, is this issue already fixed? If not I want to work in this issue

**Comment by tyrasd** [comment]
> Hello Friends and folks, is this issue already fixed? If not I want to work in this issue

no, it has not yet been implemented. go ahead @KishorKumarParoi

**Comment by nc1337** [comment]
I will try to fix this issue if it's not fixed.

**Comment by Moeinmn** [comment]
Hey , Can I work on this issue ? @tyrasd

**Comment by KishorKumarParoi** [comment]
@Moeinmn can we collaborate together, actually I am very new to solving open source projects.

**Comment by Moeinmn** [comment]
@KishorKumarParoi yes why not , i'm new to opensource too

**Comment by N-45div** [comment]
@KishorKumarParoi @Moeinmn can I also collaborate , actually I am also  very new to solving open source projects.

**Comment by shreya024** [comment]
@RudyTheDev I would like to work on this issue.Kindly assign

**Comment by RudyTheDev** [comment]
> @RudyTheDev I would like to work on this issue.Kindly assign

I don't have the ability to assign anyone to tasks/issues in this repository. I simply proposed this feature.

**Comment by tyrasd** [comment]
this was implemented in #10046
